resolwe_bio/processes/reads_processing/bqsr.py

```diff
@@ -41,8 +41,9 @@ class Input:
         bam = DataField('alignment:bam', label='BAM file containing reads')
         reference = DataField('genome:fasta', label='Reference genome file')
         known_sites = ListField(
-            DataField('variants:vcf', description='Known sites in (.vcf).'),
-            label='One or more databases of known polymorphic sites used to exclude regions around known '
+            DataField('variants:vcf', label='Known sites of variation'),
+            label='List of known sites of variation',
```

Comment:
`label` field is defined twice. Define the `ListField` as done [here](https://github.com/genialis/resolwe-bio/blob/fc51c36b636d4a396faac02285605dafe0779104/resolwe_bio/processes/differential_expression/shRNAde.py#L56)